Desktop version

gealsanchez opened the pull request:
index.html updated for the desktop version
style.css updated for the desktop version
images uploaded for the desktop version

gealsanchez committed:
index.html updated, changes for the desktop version

gealsanchez committed:
style.css updated, changes for the desktop version

gealsanchez committed:
Uploaded images for the desktop version

gealsanchez committed:
style.css updated, changes for the linters errors

gealsanchez committed:
style.css updated, changes for the linters errors

gealsanchez committed:
style.css updated, changes for the linters errors

gealsanchez committed:
style.css updated, changes for the linters errors

gealsanchez committed:
style.css updated, changes for the linters errors

frank1738 commented on index.html:
```diff
@@ -89,10 +93,10 @@ <h2 class="white-1 roboto-f f-weight700 l-height-24 f-size-26">I am a software d
 
     <div class="desktopversion">
```
- [ ] Your Desktop version appears to have the hamburger button. Kindly replace that with the desktop menu to match the original design.

![image](https://user-images.githubusercontent.com/98453979/194642208-51e7a2a5-950a-4439-bb99-321b624352be.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/194642525-359be19a-6210-4b37-9671-2abd9e4c05e6.png)

- [ ] On the project section in the original design, you will notice that only the first card has the `see project`  button but I noticed all your cards have it. Kindly fix that to match the design.

![image](https://user-images.githubusercontent.com/98453979/194643468-c1586314-fd8b-44fe-849d-e47d6877edf8.png)

- [ ] On the contact section, Kindly adjust the space between the inputs to match the Figma design. Currently, its too big

![image](https://user-images.githubusercontent.com/98453979/194644456-19c1142d-f527-4bb9-a0b4-786e8a40180b.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/194644545-585396c7-d92f-4cb2-a4f6-8efb05fa62d3.png)

frank1738 reviewed (changes requested):
Hi @gealsanchez , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


![YesYouAreGIF (2)](https://user-images.githubusercontent.com/50941074/172881298-64e98170-87c5-4075-8a77-71d65d991f55.gif)


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good Design 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gealsanchez committed:
index.html updated, code review issue, button see project, works container

gealsanchez committed:
style.css updated, code review issue, button see project, works container

gealsanchez committed:
style.css updated, code review issue, button see project, works container

gealsanchez committed:
Update style.css

no-missing-end-of-source-newline

gealsanchez committed:
Update style.css

no-missing-end-of-source-newline

gealsanchez committed:
Update style.css

Expected newline before "}" of a multi-line block   block-closing-brace-newline-before

gealsanchez committed:
Update style.css

Code Review Issue, desktop menu

gealsanchez committed:
Update index.html

Code Review Issue, desktop menu

tufoinnkuo10 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @gealsanchez 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
